pymeasure/instruments/lecroy/lecroyT3DSO1204.py

```diff
@@ -353,23 +334,65 @@ def __init__(self, instrument, number):
         map_values=True
     )
 
-    def values(self, command, **kwargs):
-        """ Reads a set of values from the instrument through the adapter,
-        passing on any key-word arguments.
-        """
-        return self.instrument.values("C%d:%s" % (self.number, command), **kwargs)
+    _measurable_parameters = ["PKPK", "MAX", "MIN", "AMPL", "TOP", "BASE", "CMEAN", "MEAN", "RMS",
+                              "CRMS", "OVSN", "FPRE", "OVSP", "RPRE", "PER", "FREQ", "PWID",
+                              "NWID", "RISE", "FALL", "WID", "DUTY", "NDUTY", "ALL"]
 
-    def ask(self, command):
-        return self.instrument.ask("C%d:%s" % (self.number, command))
+    display_parameter = Instrument.setting(
+        "PACU %s",
+        """ The waveform of this channel is processed with the specified algorithm and the result
+        is displayed on the front panel. The commands accepts the following parameters:
```

Comment:
A small suggestion:
```suggestion
        """ Set the waveform processing of this channel with the specified algorithm and the result
        is displayed on the front panel. The command accepts the following parameters:
```
We try to start all `setting` with "set" in order to tell what type of property we have.

Reply:
I fixed the docstring and I renamed the `display_parameter` property into `set_display_parameter`

Reply:
Sorry, I meant, that we start the **docstring** of a `setting` with "set", not the name of the property.

Reply:
I'll fix this locally, so we can get this in finally.

Reply:
argh, that workflow sucks with Github...

Reply:
OK, I had to create #770 and will merge over there.